addons/web_editor/static/src/js/backend/html_field.js

```diff
@@ -199,6 +200,7 @@ export class HtmlField extends Component {
                                             positionCallback: this.positionDynamicPlaceholder.bind(this),
                                         }
                                     );
+                                    this.wysiwyg.odooEditor.observerActive('openDynamicPlaceholder');
```

Comment:
So, if there is no `baseModel` the observers are never reactivated ?

Reply:
hum indeed, I'll move the observerUnactive call